src/app/mytools/my-tool/my-tool.component.ts

```diff
@@ -233,9 +170,14 @@ export class MyToolComponent extends MyEntry implements OnInit {
   }
 }
 
-export interface OrgToolObject {
+/**
+ * When using this, T should always be DockstoreTool
+ *
+ * @export
+ * @interface OrgToolObject
+ * @template T
+ */
+export interface OrgToolObject<T> extends OrgEntryObject<T> {
```

Comment:
I think you can make this `export interface OrgToolObject extends OrgEntryObject<DockstoreTool>`, then you don't need the comment on line 174, nor run the risk of someone using this interface with the wrong T.

Reply:
Nope